build.gradle

```
@@ -47,10 +47,13 @@ dependencies {
         exclude(group: 'org.openjfx')
     }
     implementation("io.github.kostaskougios:cloning:1.10.3")
-
-    testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
-    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${junitVersion}")
+    implementation 'org.openjfx:javafx-controls:11'
+    implementation 'org.openjfx:javafx-fxml:11'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
```

Change this as well and let me know if there are any errors. 
```suggestion
    implementation("org.openjfx:javafx-controls:11")
    implementation("org.openjfx:javafx-fxml:11")
    testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
```